src/components/DownloadAppModal.js

```diff
@@ -44,6 +44,8 @@ function DownloadAppModal({isAuthenticated, showDownloadAppBanner}) {
     }
 
     const handleOpenAppStore = () => {
+        setShowDownloadAppModal(false);
+        setshouldShowBanner(false);
```

Comment:
I know that it's not your implementation - but maybe you can fix this typo to be setShouldShowBanner

<img width="623" alt="Screenshot 2023-09-05 at 14 20 10" src="https://github.com/Expensify/App/assets/28352309/172b208f-4f7e-4258-9d57-826094b3f4c0">